content/site-map.yaml

```diff
@@ -301,8 +301,6 @@
               file: "/learn/guides-and-tutorials/guides-and-tutorials"
             - name: "Introduction to Terra"
               file: "/learn/intro-to-terra/intro-to-terra"
-            - name: "Introduction to Gen3"
-              file: "/learn/accessing-data/getting-started-with-gen3"
```

Comment:
Thanks! Can you delete this file as well please? 
Cheers,
D

Reply:
Is there any concern about directing to a broken link from an external page? Happy to remove if not.

Reply:
Ok we can clean that up. Thx.

Reply:
Ok, removed refs to that page!